extensions/queries/QueriesController.php

```diff
@@ -256,6 +257,10 @@ public function editorAction()
                             $contentType = 'application/json';
                             $filename = 'query-result.json';
                             break;
+                        case 'csv':
+                            $contentType = 'text/csv';
+                            $filename = 'query-result.csv';
+                            break;
                     }
```

Comment:
This is already in the latest OntoWiki: https://github.com/AKSW/OntoWiki/pull/280